SDL2/SDL2_image.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "SDL2_image",
+    "cleanup": [
+        "/include",
+        "/lib64/cmake",
+        "/lib64/pkgconfig"
```

Comment:
```suggestion
        "/lib/cmake",
        "/lib/pkgconfig"
```

Reply:
I tested it locally, and the cmake build puts these in `lib64`.